### PR TITLE
Add installation instructions to the docmentation

### PR DESCRIPTION
**Pull request by DunklesArchipel**
Checklist
* [x] Added an entry in `doc/news/`. 
* [X] Added a test for this change. --> not required
* [x] Adapted the documentation for this change. --> not required

Adds installation instructions. These are copy paste from svgdigitizer, replacing svgdigitizer.

Addresses #26

### CODE REVIEW

File: doc/installation.md [hunk 1]
```diff
@@ -1,9 +1,95 @@
...
+
+```sh
+pip install --user echemdb
 ```
+
+You can instead also install the latest unreleased version of the echemdb
+from our [GitHub Repository](https://github.com/echemdb/echemdb) with
+
+```sh
+pip install git+https://github.com/echemdb/echemdb@master
```
Comment (saraedum) [inline doc/installation.md]:
```suggestion
pip install git+https://github.com/echemdb/echemdb@main
```

File: doc/installation.md [hunk 2]
```diff
@@ -1,9 +1,95 @@
...
+
+```sh
+conda create -n echemdb echemdb
+```
+
+To use the echemdb, activate the `echemdb` environment:
+
+```sh
+conda activate echemdb
+echemdb --help
```
Comment (saraedum) [inline doc/installation.md]: Is there an `echemdb` command?

**Review by saraedum — APPROVED** [review]

File: doc/installation.md (hunk 2 above)
Comment (DunklesArchipel) [inline doc/installation.md]:
```suggestion
```

Comment (DunklesArchipel) [reply doc/installation.md]: right this will not work

**Review by saraedum — APPROVED** [review]

**Pull request merged**